### PR TITLE
MC-11761 Adds delete workload documentation

### PR DESCRIPTION
**Pull request by m-orfanos**
#### Changes made

- Adds delete workload (daemon sets, deployments and stateful sets) documentation
- Use tool to auto-format documents

### CODE REVIEW

**Comment by m-orfanos** [comment]
~Sorry for all the whitespace changes 😬~

Yeah, I removed them.

File: source/includes/kubernetes/_k8_deployments.md [hunk 1]
```diff
@@ -61,34 +61,35 @@ curl -X GET \
 
 ```json
 {
-    "data": {
-      "id": "apm-server-1585600296-apm-server/monitoring",
-      "deploymentStatus": "Available",
-      "readyRatio": "1/1",
-      "metadata": {},
```
Comment (jasondias9) [inline source/includes/kubernetes/_k8_deployments.md]: I know this isn't related to this PR necessarily but should we be including examples of the nested fields of the metadata, spec in the example response. These are typically never empty

**Comment by m-orfanos** [comment]
I reverted all the whitespace changes, there were way too many.

**Review by jasondias9 — APPROVED** [review]

File: source/includes/kubernetes/_k8_deployments.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/kubernetes/_k8_deployments.md]: The problem with adding examples for nested attributes would be the API docs would consist of mostly json objects. We refer them to the Kubernetes API docs instead.

**Review by y0geshdev — APPROVED** [review]

**Pull request merged**